test/e2e/storage/csi_volumes.go

```diff
@@ -48,6 +49,8 @@ var csiTestDrivers = map[string]func(f *framework.Framework, config framework.Vo
 	"hostPath": initCSIHostpath,
 	// Feature tag to skip test in CI, pending fix of #62237
 	"[Feature: GCE PD CSI Plugin] gcePD": initCSIgcePD,
+	"[Feature:Volumes] rbd":              initCSIrbd,
```

Comment:
"[Feature:Volumes] rbd csi driver" (so we can just run csi driver explicitly in e2e test)

Reply:
Fixed like "csi xxx" instead "xxx csi driver", because the test names tend to be long.